src/Plugin.php

```diff
@@ -116,6 +116,15 @@ public static function renderTemplate(array $template_subpathnames, array $varia
     }
   }
 
+  /**
+   * Checks if wp-rocket plugin is active and images lazyload option is set.
+   *
+   * @return bool
+   */
+  public static function lazyLoadIsActive() {
```

Comment:
Methods that result in Yes or No answers should always start with `is`, i.e., `isLazyLoadEnabled()`